### ISSUE TITLE
Add picture of taiga village on curseforge/pmc/modrinth

### ISSUE THREAD

**Issue by vinnyg0621** [jtorleon-studios-team/bettervillages#127]
it's the new update but I've discovered that you're not giving credit to the newest and coolest village of the mod on the mod's webpage??? that is just so disappointing! or, if people haven't installed this mod yet and have **not** even seen the taiga village in the mod in the brand-new update, then there's no spruce photos??! **wait!** you showed off photos on the curse page of the savannah village when it first came out but not TAIGA??!! c'mon, you're smarter than that!
please add taiga village photos on curseforge thank you! :D

**Comment by tmslpm** [comment]
You're right, as soon as we have the time, we'll take beautiful pictures with the best complementary shaders!
Added to board

**Comment by jtl-elisa** [comment]
Done.  Image added to curseforge project

![Image](https://github.com/user-attachments/assets/20d82597-8787-4ec4-b4d2-beaefd1973b6)
![Image](https://github.com/user-attachments/assets/97677428-da47-4b96-9945-d31da5a12a86)
![Image](https://github.com/user-attachments/assets/bc16041d-cb6e-423a-82df-3f7acb5a65a9)
from: https://discord.com/channels/1295052035976335420/1295368037083385916/1340469343389089832
